mangum/handlers/aws_alb.py

```diff
@@ -62,7 +62,9 @@ def encode_query_string(self) -> bytes:
         Issue: https://github.com/jordaneremieff/mangum/issues/178
         """
 
-        params = self.trigger_event.get("multiValueQueryStringParameters")
+        params: Optional[
```

Comment:
This might be better to create a new [type alias](https://docs.python.org/3/library/typing.html#type-aliases) because it seems difficult to read - consider this optional and not required to merge.